roxctl/central/crs/list.go

```diff
@@ -0,0 +1,72 @@
+package crs
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"text/tabwriter"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	pkgCommon "github.com/stackrox/rox/pkg/roxctl/common"
+	"github.com/stackrox/rox/pkg/utils"
+	"github.com/stackrox/rox/roxctl/common"
+	"github.com/stackrox/rox/roxctl/common/environment"
+	"github.com/stackrox/rox/roxctl/common/flags"
+	"github.com/stackrox/rox/roxctl/common/util"
+)
+
+func listCRSs(cliEnvironment environment.Environment, timeout time.Duration, retryTimeout time.Duration) error {
+	ctx, cancel := context.WithTimeout(pkgCommon.Context(), timeout)
+	defer cancel()
+
+	conn, err := cliEnvironment.GRPCConnection(common.WithRetryTimeout(retryTimeout))
+	if err != nil {
+		return err
+	}
+	defer utils.IgnoreError(conn.Close)
+	svc := v1.NewClusterInitServiceClient(conn)
+
+	tabWriter := tabwriter.NewWriter(cliEnvironment.InputOutput().Out(), 4, 8, 2, '\t', 0)
```

Comment:
nitpick: we can move this just before the first `fmt.Fprintln(tabWriter`.

Reply:
done